scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -721,7 +719,8 @@ sealed trait TypedPipe[+T] extends Serializable {
     delta: Double = 0.01, //5 rows (= 5 hashes)
     seed: Int = 12345)(implicit ev: TypedPipe[T] <:< TypedPipe[(K, V)],
       serialization: K => Array[Byte],
-      ordering: Ordering[K]): Sketched[K, V] =
+      ordering: Ordering[K],
+      m: Quoted): Sketched[K, V] =
     Sketched(ev(this), reducers, delta, eps, seed)
```

Comment:
Can we pass `m` explicitly to `Sketched` so we can see that it is not discarded?

Reply:
fixed